src/test/java/org/jabref/model/database/event/AutosaveEventTest.java

```diff
@@ -0,0 +1,14 @@
+package org.jabref.model.database.event;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class AutosaveEventTest {
+
+    @Test
+    public void givenNothingWhenCreatingThenNotNull() {
```

Comment:
What is this test useful for? When creating a java object, the object is always non-null.

Maybe, it could be check for no exceptions being thrown. Maybe add `AndNoExceptionThrown`.

Reply:
Good point, I will add another test for exceptions.

Yes, the tests concern exception handling. While trivial, they may turn out for regression tests later. In general, I would like to increase overall testcoverage of classes, then methods, then lines. 

I start with class covering, which sometimes is trival, but allows me to get used to this projects conventions.